docs/release_notes/v1.10.2.md

```diff
@@ -0,0 +1,20 @@
+# Dapr 1.10.2
+
+## Fix removing extra commits from master branch in 1.10.1
+
+### Problem
+
+This release does not address any known issue. The Dapr 1.10.1 hotfix contained additional commits from the `master` branch that were not part of the 1.10 release branch.
+The 1.10.2 hotfix guarantees that all changes in the release are tested and validated.
+
+### Impact
+
+There is no known impact to users.
```

Comment:
Shouldn’t we say here that there is an impact as the build contained code that hadn’t been thoroughly tested yet?